en/docs/develop/product-apis/admin-apis/admin-v0.16/admin-v0.16.yaml

```diff
@@ -1704,6 +1704,68 @@ paths:
 
 
 
+######################################################
+# The "Application Collection" resource APIs
+######################################################
+  /applications/{applicationId}:
+
+#-----------------------------------------------------
+# Retrieve an application by application id
+#-----------------------------------------------------
+    get:
+      x-scope: apim:app_owner_change
+      x-wso2-request: |
+        GET https://localhost:9443/api/am/admin/v0.16/applications/890a4f4d-09eb-4877-a323-57f6ce2ed79b
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8\" \"https://localhost:9443/api/am/admin/v0.16/applications/890a4f4d-09eb-4877-a323-57f6ce2ed79b\""
+      x-wso2-response: "HTTP/1.1 200 OK\nContent-Type: application/json\n\n{\n         \"groupId\": \"\",\n         \"owner\": \"admin\",\n           \"applicationId\": \"890a4f4d-09eb-4877-a323-57f6ce2ed79b\",\n         \"status\": \"APPROVED\",\n         \"name\": \"app1\"\n}"
```

Comment:
Please use x-code-samples instead of these. Can refer to other resources added here.